hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieInternalConfig.java

```diff
@@ -52,6 +52,12 @@ public class HoodieInternalConfig extends HoodieConfig {
       .markAdvanced()
       .withDocumentation("Inner configure to pass static partition paths to executors for SQL operations.");
 
+  public static final ConfigProperty<WriteOperationType> BULK_INSERT_WRITE_OPERATION_TYPE = ConfigProperty
```

Comment:
Can we eliminate it, it does not make sense to introduce a sub-type and a default value is itself.

Reply:
> Can we eliminate it, it does not make sense to introduce a sub-type and a default value is itself.

Done~